annotate.el

```diff
@@ -164,7 +164,7 @@ file will be shown."
   :type 'boolean
   :group 'annotate)
 
-(defcustom annotate-annotation-confirm-deletion t
+(defcustom annotate-annotation-confirm-deletion nil
```

Comment:
I wonder if it is even necessary to implement deletion-confirmation at all if we're disabling it by default anyway. I'll leave that up to you, though.